_posts/2023-11-29-cockpit-306.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Cockpit 306
+author: pitti
+date: '2023-11-29'
+tags: cockpit, machines, podman, ostree
+slug: cockpit-306
+category: release
+summary: 'kdump automation, OSTree redesign, deleting intermediate podman images'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 306, cockpit-machines 303, cockpit-podman 82, and cockpit-ostree 198:
+
+
+## Kdump: Add Ansible/shell automation
+
+The Kdump page can generate an [Ansible](https://www.ansible.com/automation) role or a shell script for replicating the current kdump configuration to other machines.
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation.png)
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation-2.png)
+
+## OSTree: Redesign with new features: status, clean up, reset, and pin
```

Comment:
This is too long, it's a bit ugly when the title wraps to two lines. @garrett: Perhaps just "OSTree: Redesign with new features" and we leave the details to the paragraphs?

Reply:
Sure!

```suggestion
## OSTree: Redesign, with new features
```